tests/security/selinux/semodule.pm

```diff
@@ -25,7 +25,7 @@ use utils;
 
 sub run {
     my ($self) = @_;
-    my $test_module = "apache";
+    my $test_module = "openvpn";
```

Comment:
Just curious why we change the $test_module from apache to openvpn ?

Reply:
"apache" module has some dependency modules I have to handle with new policy pkgs.
So I changed to a easy one.

Reply:
OKAY. I see. LGTM. :)